frontend/server/src/Controllers/User.php

```diff
@@ -8,17 +8,22 @@
  * @psalm-type PageItem=array{class: string, label: string, page: int, url?: string}
  * @psalm-type AuthorsRank=array{ranking: list<array{author_ranking: int|null, author_score: float, classname: string, country_id: null|string, name: null|string, username: string}>, total: int}
  * @psalm-type AuthorRankTablePayload=array{length: int, page: int, ranking: AuthorsRank, pagerItems: list<PageItem>}
+ * @psalm-type Badge=array{assignation_time?: \OmegaUp\Timestamp|null, badge_alias: string, unlocked?: boolean, first_assignation?: \OmegaUp\Timestamp|null, total_users?: int, owners_count?: int}
```

Comment:
```suggestion
 * @psalm-type Badge=array{assignation_time?: \OmegaUp\Timestamp|null, badge_alias: string, unlocked?: bool, first_assignation?: \OmegaUp\Timestamp|null, total_users?: int, owners_count?: int}
```
Desconozco si sea válido usar `boolean`, pero por consistencia es mejor usar `bool` aquí en psalm. Esto posiblemente ocasione que las pruebas fallen, porque hay otro lugar donde se declara `Badge` y los dos deben de ser idénticos, así que también habría que corregirlo